Sources/SwiftGRPCNIO/GRPCStatus.swift

```diff
@@ -6,11 +6,11 @@ public struct GRPCStatus: Error {
   /// The code to return in the `grpc-status` header.
   public let code: StatusCode
   /// The message to return in the `grpc-message` header.
-  public let message: String
+  public let message: String?
   /// Additional HTTP headers to return in the trailers.
   public let trailingMetadata: HTTPHeaders
 
-  public init(code: StatusCode, message: String, trailingMetadata: HTTPHeaders = HTTPHeaders()) {
+  public init(code: StatusCode, message: String? = nil, trailingMetadata: HTTPHeaders = HTTPHeaders()) {
```

Comment:
I'd rather avoid the ` = nil` default here; users should be encouraged to always provide a message.